### ISSUE TITLE
AN Escape 3 Dial tone doesn't play when coin is inserted

### ISSUE THREAD

**Issue by guyman70718** [ruffle-rs/ruffle#1620]
Error Info
Look at floor, lift rug, grab coin. Look at phonebooth table, grab string. Click string and click it on top of the coin in your inventory. Click the phone and lift it, then insert the coin on a string. A dial tone should start to play.

Game url: [](http://www.afro-ninja.com/game/47)

Ruffle Info

Ruffle version: nightly 11/20/20
Ruffle source: desktop and web demo

**Comment by Toad06** [comment]
Seems like you forgot to post the URL. 😛

**Comment by guyman70718** [comment]
Oh, duh. Thanks haha. https://www.newgrounds.com/portal/view/411079

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Toad06** [comment]
The dial tone is supposed to play once the coin effect has finished playing, but the `Sound.onSoundComplete` listener is currently not implemented in Ruffle (cc #273 and #396), so this never happens.

```
coindeposit.onSoundComplete = function(Void)
{
	if(_root.phoneStatus == "moneyDeposit")
	{
		_root.phoneStatus = "dialTone";
		_root.dialtone.start();
	}
};
```

**Comment by Herschel** [comment]
Fixed by #2825.
